core/src/autogluon/core/metrics/__init__.py

```diff
@@ -454,8 +454,11 @@ def local_pearsonr(y_true, y_pred):
                        greater_is_better=True)
 
 
-def rmse_func(y_true, y_pred):
-    return np.sqrt(((y_true - y_pred) ** 2).mean())
+def rmse_func(y_true, y_pred, **kwargs):
+    if kwargs:
+        return sklearn.metrics.mean_squared_error(y_true, y_pred, squared=False, **kwargs)
```

Comment:
Why not use this directly and drop numpy version ?

Reply:
Discussed with Nick, following the implementation of roc_auc, we keep this to use accelerated (skipping validation checks) implementation if no additional argument is given